distribution/archives/integ-test-zip/src/test/java/org/elasticsearch/test/rest/CreatedLocationHeaderIT.java

```diff
@@ -46,7 +46,7 @@ public void testIndexWithoutId() throws IOException {
     }
```

Comment:
^^ methods testCreate, testIndexWithId and testIndexWithoutId still going with "test" doc type. Is this intentional?

Reply:
It wasn't intentional. I switched them to `_doc` when deprecating get requests, so this should be fixed.